pages/docs/tutorials/streetlights.md

```diff
@@ -1,244 +1,263 @@
-
-We would love to see what you create with AsyncAPI! As an open-source project, we're open to proposals, questions, suggestions, and contributions. If you don't feel in the mood to contribute but you're using AsyncAPI, just raise your hand by [creating an issue in our Github repo](https://github.com/asyncapi/asyncapi/issues/new) or [join our Slack channel](https://www.asyncapi.com/slack-invite/). Don't be shy.
-
----
-
-<DocsButton
-  suggestions={[
-    {
-      href: '/docs/tutorials',
-      title: 'Tutorials - Overview',
-      type:'back',
-    },
-    {
-      href: '/docs/tutorials/streetlights-interactive',
-      title: 'Streetlights - Interactive (Alpha)',
-      type:'next',
-    }
-  ]}
-/>
\ No newline at end of file
+---
+title: Streetlights
+description: In this tutorial, you'll get started with code samples from an IoT real-world use case. Let's pretend you have a company called Smarty Lighting that engineers smart-city lighting systems.
+---
+
+# Introduction
+
+In this tutorial, you get started with actual code and a sample real-world use case. Let's pretend you have a company called Smarty Lighting and you install smart-city streetlight lighting systems. This is a use case of Internet of things (IoT).
+
+We are going to use Node.js and Mosquitto as the message broker to build a Smartylighting Streetlights API which allows you to manage city lights remotely.
+
+You want to build a system that can turn streetlights on and off based on their environmental conditions: 
+
+- You're going to implement an event-driven architecture (EDA), with a Message Broker in its "center".
+- Streetlights will publish information about their environmental lighting to the broker.
+- Your application will connect to the broker and receive a stream of events from all the streetlights that are reporting their conditions.
+- Based off the events, your application can make a decision regarding turning the streetlight off.
+- Your application is not aware of how many streetlights are publishing events - it just connects to the broker and receives all events
+
+You will learn about event-driven architecture, message broker,pub/sub architecture, create an AsyncAPI file to describe your API and generate code from it.
+
+# Background Context
+
+Event-driven architecture (EDA) is a design pattern built around the production, detection, and reaction to events that take place in time.
+[Message brokers](https://deploy-preview-601--asyncapi-website.netlify.app/docs/tutorials/getting-started/event-driven-architectures#message-broker) enable asynchronous communications between services so that the sending service need not wait for the receiving service’s reply.This allows interdependent services to “talk” with one another directly, even if they were written in different languages or implemented on different platforms. 

```

Comment:
```suggestion
[Message brokers](https://deploy-preview-601--asyncapi-website.netlify.app/docs/tutorials/getting-started/event-driven-architectures#message-broker) enable asynchronous communications between services so that the sending service need not wait for the receiving service’s reply. This allows interdependent services to “talk” with one another directly, even if they were written in different languages or implemented on different platforms. 

```